pkg/cli/admin/copytonode/command.go

```diff
@@ -21,10 +21,7 @@ var (
 
 	copyToNodeExample = templates.Examples(`
 		# copy a new bootstrap kubeconfig file to every node
```

Comment:
we should also update the annotation here.

Reply:
Example with the annotation removed altogether.